.github/workflows/website.yml

```diff
@@ -0,0 +1,60 @@
+name: "Update website from a GitHub release of GAP"
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Release tag name (e.g. v4.12.1; leave empty to use the latest release)"
+        required: false
+      date:
+        description: "YYYY-MM-DD format GAP release date (leave empty to use the date in the GAP release tarball)"
+        required: false
+  # TODO delete `on: push`
+  # The `on: push` trigger is temporary.
+  # You can only manually dispatch workflows that exist in the master branch,
+  # and this Workflow doesn't yet exist in the master branch.
+  push:
+    branches:
+      - website-scripts
+
+jobs:
+  website:
+    name: "Update from GAP release"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check out the GapWWW repository in order to make changes to it"
+        uses: actions/checkout@v2
+      - name: "Clone GAP for its most up-to-date release scripts"
+        uses: actions/checkout@v2
+        with:
+          # TODO stop using wilfwilson/gap@website-scripts once it is merged
+          #repository: gap-system/gap
+          #ref: master
+          repository: wilfwilson/gap
+          ref: ww/website-scripts
+          path: tmp/gap/
+          fetch-depth: 1
+      - uses: actions/setup-python@v2
+      - name: "Install the required Python modules"
+        run: pip3 install PyGithub requests python-dateutil
+      - name: "Run the GAP release script for updating the website"
+        id: gap
+        run: |
+          # TODO pass on inputs.tag and inputs.date
+          python -u ./tmp/gap/dev/releases/update_website.py
+          echo "::set-output name=version::$(grep version: _data/release.yml | cut -d\' -f2)"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: "Make a pull request to the master branch of this repository"
+        uses: peter-evans/create-pull-request@v3
```

Comment:
Oh, nice, it creates a PR. I like it!

Side remark: I'd like to eventually migrate the package distribution mechanism to Github Actions, too; my idea was to keep all those packages (resp. package snapshots) in a single git repo; and when a new package release is made, then a cron job create a PR adding that package; and then CI tests for that package are run. But package authors can also submit manual PRs, e.g. if multiple packages need to be updated simultaneously.

And this is kinda one of the missing pieces for that... So I am getting more confident that the above idea can be implemented with reasonable effort.

Anyway: Thanks for this PR. Let me know if/when you want a review (but I think there is not too much I can review here right now, we just need to merge and test it at some point?)

Reply:
That would be good; the website could then use that repository to then always know what are the latest versions of packages (i.e. beyond those included in the latest GAP release).

I'm in the middle of making further changes to GapWWW, and to GAP's `dev/releases/update-website.py` script. Once they're both finished (and in sync again) then I'll rebase this PR and it'll be ready for review/merge; basically all that this PR does it run `dev/releases/update-website.py` and create a PR with the changes.